dask/dataframe/tests/test_rolling.py

```diff
@@ -270,7 +270,10 @@ def test_time_rolling_constructor():
 @pytest.mark.parametrize("window", ["1S", "2S", "3S", pd.offsets.Second(5)])
 def test_time_rolling_methods(method, args, window, check_less_precise):
     if dd._compat.PANDAS_GT_110:
-        check_less_precise = {}
+                if check_less_precise:
```

Comment:
Oof, sorry, I messed up the indentation in my suggestion 😄. Black might complain.

Reply:
black did :)

Reply:
I didn't even look at the diff :roll_eyes: